include/xtensor-io/xhighfive.hpp

```diff
@@ -118,7 +130,7 @@ namespace xt
         {
             if (!exist(file, path))
             {
-                throw std::runtime_error("xt::extensions::size: Field does not exist ('"+path+"')");
+                detail::error(file, path, "xt::extensions::size: Field does not exist");
```

Comment:
I wonder if there is a difference of `throw`ing here vs throwing in the helper function? 

`detail::error` could also just return an instance of std::runtime_error that get's thrown at this point here.

Reply:
Not sure either, but it's a good idea.